### ISSUE TITLE
Bug: deSEC.io provider support for "*" / wildcard subdomains

### ISSUE THREAD

**Issue by mihakrumpestar** [qdm12/ddns-updater#572]
**TLDR**: *deSEC.io provider support for "\*" / wildcard subdomains*

1. Is this urgent: No, but it is important
2. DNS provider(s) you use: deSEC
3. Program version: 

```
qmcgaw/ddns-updater:latest

org.opencontainers.image.created 	2023-11-17T14:08:09.354Z
sha256:fc65d8f5588a6b25a4021bb0b2c1de94ea42d4a96b9be705edeb1a2183c8352d
```

4. What are you using to run the container: docker-compose
5. Extra information: 

This is listed as bug and not feature, as it works as per the deSEC docs.

1. As per [deSEC docs](https://desec.readthedocs.io/en/latest/dns/rrsets.html#rrset-field-reference) and [here](https://desec.readthedocs.io/en/latest/dns/rrsets.html#modifying-an-rrset) this (updating wildcard subdomains) should not be a problem, so why is this disabled in code?

2. I am not sure why this is disabled for most providers, as wildcard domains are nothing special.

3. I am asking this as I went through the repo, docs, issues and GitHub search, but could not find any answer (If I missed it, please correct me.). I am sure that many use "*" subdomains, so I am surprised that no one raised concerns before (except [here](https://github.com/qdm12/ddns-updater/issues/537)).

Please, don't get me wrong, it is a great project, but this thing has been really bugging me.

**Comment by qdm12** [comment]
1. Indeed, I'm not sure why I disabled it for desec; the code is using the (simpler) dyndns API, I removed the limitation on wildcards in commit 37f473160cc417e9a206386eadca69f4197998d0 (latest image built today), please try and let me know if it works 😉 If it doesn't I can change the code to use their more complex rest API, but ideally not.
2. Quite a few providers error out when trying to update a wildcard host (especially using dyndns format). I created issue #593 to track this, I'll investigate later each of the providers.
3. See 2. 😉 I am working on bug fixes to have a (long overdued) 2.6 release first, and then I'll get to the wildcards!

> Please, don't get me wrong, it is a great project, but this thing has been really bugging me.

Thanks & this makes sense 👍 !

**Comment by mihakrumpestar** [comment]
Tested the wildcard and it works as it should. 
Thank you. 

I do owe an apology, as I should have tested this before and prepared a pull request. I should have been more proactive.

**Comment by qdm12** [comment]
I'm glad you mentioned the wildcard should be working! 👍 One more important feature with two less lines of code, I call this a win 😄 

> I should have been more proactive.

Tell me about it, I replied 3 weeks later 😆 No worry about it, life gets in the way and forking/cloning/branching-out/etc. is quite a few steps to change 2 lines 🤣 

Closing this 😉
